ibtk/include/ibtk/LDataManager.h

```diff
@@ -86,7 +86,90 @@ namespace IBTK
  * for all inter-processor communications.  All access to instantiated
  * LDataManager objects is via the static method getManager().
  *
+ * Each Lagrangian point is associated with an LNode object which provides an interface between the Lagrangian and PETSc
+ * ordering as well as the data storage for forces and objects assosciated with the point. From each LNode object, we
```

Comment:
Do you mean something like:

```suggestion
 * ordering as well as the data storage for force specification objects and other objects associated with the point. From each LNode object, we
```

Resultant forces are stored in PETSc `Vec`s.